pkg/postgres/pgadmin/admin_utils.go

```diff
@@ -76,6 +82,13 @@ func CreateDB(sourceMap map[string]string, adminConfig *pgxpool.Config, dbTempla
 
 	SetPostgresCmdEnv(cmd, sourceMap, adminConfig)
 
+	// terminate connections to the source database.  You cannot copy from a database if
+	// there are open connections to it.
+	err := terminateConnection(adminConfig, dbTemplate)
```

Comment:
Since we have the pool that is currently active, any reason we shouldn't call close on it? This will prevent new connections from being started right?

Reply:
Also maybe we could throw back an error if someone is logged in despite this call?

Reply:
@connorgorman 
That begs a question of whether I'm over protecting us here.  When we make a copy of a non-empty database we are running exclusively from the migrator.  Central has not created its pool at that time.  In the migrator we create pools when we need them and close them because we are moving stuff around.  Where I've seen this issue and what I was trying to protect itself is the unknown connection.  For example, this issue occurred in the demo because I was in psql.  That is the only time I've seen it and what I was trying to protect against.  I do not we can have a connection to a database we are copying from through execution of the code, as it is written today.  So it seemed like a reasonable thing to do, but is it a valid scenario?